.azuredevops/pipelineTemplates/module.jobs.deploy.yml

```diff
@@ -231,6 +231,7 @@ jobs:
           #---------------------
           - task: AzurePowerShell@5
             displayName: 'Deploy [${{ deploymentBlock.path }}] via connection [${{ parameters.serviceConnection }}]'
+            name: 'DeployModule'
```

Comment:
Any reason to add the 'id/name'?

Reply:
It's not referenced

Reply:
The id is needed to allow accessing the output from another stage. E.g.

```
- stage: stageB
  dependsOn: stageA
  jobs:
  - job: stageBjob
    variables:
        varFromStageA: $[ stageDependencies.stageA.stageAjob.outputs['DeployModule.deploymentOutput'] ]
```